app/controllers/buyers/users_controller.rb

```diff
@@ -92,7 +92,7 @@ def user_params
   end
 
   def redirect_back_or_show_detail
-    redirect_to(:back)
+    redirect_back(fallback_location: root_path)
   rescue ActionController::RedirectBackError
     redirect_to admin_buyers_account_user_path(account_id: user.account_id, id: user.id)
```

Comment:
I think you should merge those lines

Reply:
@hery I tried but it fails the test so I redirected it to root_path and which makes test passed.

Reply:
What test is failing?
The rescue is the same as fallback ...

Reply:
I tried with redirect_to but I think If I give this path as fallback then it will not fail and we can remove rescue .

Reply:
@hallelujah Updated location